tests/cases/widgets/menubar.slint

```diff
@@ -0,0 +1,68 @@
+
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+import { AboutSlint, Button } from "std-widgets.slint";
+export component TestCase inherits Window {
+    MenuBar {
+        entries: [
+            { title: "File" },
+            { title: "Edit" },
+        ];
+        sub-menu(entry) => {
```

Comment:
My feeling is that we should keep this API private (by the time we stabilise this). Also, the string based sub-menu collection will be fun when menu text is translated :)